organisms/organisation-unit-tree/new-features/context-menu.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Org unit node context menu
+
+  Scenario: The user hovers a not disabled org unit
+    Given the user hovers an org unit
+    When the org unit is not disabled
+    When the user clicks on the context menu icon
+    Then a context menu should open
+    And the context menu is anchored to the the context menu icon
+    And the context menu icon displays in an active state
+
+  Scenario: The context menu icon is hidden by default
+    Given an org unit is displayed
```

Comment:
I think the two "Given" and "When" sentences can be combined:
`Given the user does not hover an org unit`

the `When` sentences are not mandatory, `Given` and `Then` can be totally fine

Reply:
Ah, I was under the impression `when` was mandatory. Thanks for clearing that up. I've changed this to use your suggestion.